setup.cfg

```diff
@@ -49,6 +51,7 @@ install_requires =
     python-dateutil;  python_version < "3.7"
     semantic-version
     tqdm
+    typing_extensions;  python_version < "3.8"
```

Comment:
<details>
<summary>I quickly checked -- none of those seems to be a heavy dependency so ok to just add</summary> 

```shell
lena:/tmp
$> py=3.7; d=venvs/dev$py; rm -rf $d; virtualenv --python=python$py $d && source $d/bin/activate && pip3 install pydantic email-validator typing_extensions
created virtual environment CPython3.7.7.final.0-64 in 274ms
  creator CPython3Posix(dest=/tmp/venvs/dev3.7, clear=False, global=False)
  seeder FromAppData(download=False, chardet=latest, progress=latest, distlib=latest, contextlib2=latest, colorama=latest, appdirs=latest, pip=latest, urllib3=latest, CacheControl=latest, setuptools=latest, lockfile=latest, distro=latest, pkg_resources=latest, pytoml=latest, idna=latest, html5lib=latest, wheel=latest, pyparsing=latest, msgpack=latest, certifi=latest, webencodings=latest, pep517=latest, packaging=latest, six=latest, retrying=latest, requests=latest, via=copy, app_data_dir=/home/yoh/.local/share/virtualenv/seed-app-data/v1.0.1.debian)
  activators BashActivator,CShellActivator,FishActivator,PowerShellActivator,PythonActivator,XonshActivator
Collecting pydantic
  Downloading pydantic-1.6.1-cp37-cp37m-manylinux2014_x86_64.whl (8.6 MB)
     |████████████████████████████████| 8.6 MB 4.5 MB/s 
Collecting email-validator
  Using cached email_validator-1.1.1-py2.py3-none-any.whl (17 kB)
Collecting typing_extensions
  Using cached typing_extensions-3.7.4.3-py3-none-any.whl (22 kB)
Requirement already satisfied: idna>=2.0.0 in ./venvs/dev3.7/lib/python3.7/site-packages (from email-validator) (2.8)
Collecting dnspython>=1.15.0
  Using cached dnspython-2.0.0-py3-none-any.whl (208 kB)
Installing collected packages: pydantic, dnspython, email-validator, typing-extensions
Successfully installed dnspython-2.0.0 email-validator-1.1.1 pydantic-1.6.1 typing-extensions-3.7.4.3

```
</details>